openupgrade_scripts/scripts/product/18.0.1.2/pre-migration.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2025 Hunki Enterprises BV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade
+
+xmlid_renames = [
+    ("product.group_discount_per_so_line", "sale.group_discount_per_so_line"),
+]
+
+column_creates = [
+    ("product_attribute", "active", "boolean", "TRUE"),
+    ("product_attribute_value", "active", "boolean", "TRUE"),
+    ("product_pricelist_item", "display_applied_on", "varchar", "'1_product'"),
+]
+
+
+def create_columns(env, column_spec):
```

Comment:
probably makes sense to add this to openupgradelib if we want to precreate all columns with default values?

Reply:
🤷‍♂️ Well, it already exists `add_fields`. But it is a more complex method, which creates not only the column but the field too, and if it isn't necessary, I prefer to avoid it in this kind of cases, as they are added with an OU description.

Reply:
Is it already merged :)